main.tf

```
@@ -4,7 +4,7 @@ terraform {
 
 module "ied-lambda-graphql" {
   source  = "app.terraform.io/ied/lambda-graphql/aws"
-  version = "~>1.0.0"
+  version = "~>1.0.1"
```

This is more a feature than a fix, no? 🤔 
```suggestion
  version = "~>1.1.0"
```

---

sure it is